functions/Element/getElementPosition.yaml

```diff
@@ -0,0 +1,30 @@
+shared:
+  name: 'getElementPosition'
+  pair: 'setElementPosition'
+  description: |
+    This function allows you to retrieve the position coordinates of an element. This can be any real world element, including:
+    - Players
+    - Vehicles
+    - Objects
+    - Pickups
+    - Markers
+    - Collision shapes
+    - Blips
+    - Radar areas
```

Comment:
Convert each one to a pseudo-link like `[[player|Players]]` that we can replace later.

Reply:
In wiki they're with first capital letter (`* [[Element/Player|Players]]`), but i'll do 'player' instead of 'Player' as you commented